### PR TITLE
added prop for showing value above bar 

### PR DESCRIPTION
**Pull request by jorgnyg**
reused most of the logic from renderBarTops. 

Example of result
image

### CODE REVIEW

File: src/bar-chart.js [hunk 1]
```diff
@@ -73,25 +98,36 @@ class BarChart extends AbstractChart {
       horizontalLabelRotation = 0,
       withInnerLines = true,
       showBarTops = true,
-      segments = 4
+      showTextTop = false,
```
Comment (Hermanya) [inline src/bar-chart.js:101]: could you change the prop name to something more like `showValuesOnTopOfBars`, and add it to Readme and index.d.ts

Comment (jorgnyg) [reply src/bar-chart.js]: Please check latest commit. should fix what you ask for:)

**Comment by Hermanya** [comment]
Thank you for your contribution and sorry for the delay :)
This is now available in v5.6.0

**Comment by mlopezfernandez** [comment]
This doesn't work in last version. I used npm to install package instead of yarn but the prop doesn't exist in BarChart. Other friend tried the same using yarn but same result.

**Comment by jorgnyg** [comment]
@mlopezfernandez weird..
If you add `"react-native-chart-kit": "https://github.com/indiespirit/react-native-chart-kit.git"` to the dependencies and do `npm i`, it will use the latest version.

**Comment by tavareshenrique** [comment]
Is it possible to format these values at the top of the bars using something like AxisLabel?

**Pull request merged**